charts/postgres-operator/crds/operatorconfigurations.yaml

```diff
@@ -324,8 +328,17 @@ spec:
                     type: integer
                   spilo_runasgroup:
                     type: integer
+                  spilo_runasnonroot:
```

Comment:
Why do you need this if there is a `spilo_runasuser`?

Reply:
Such field is required to conform with pod security standards restricted profile

Reply:
Ok then let's have it!
Please resolve all the comments related with this one.
Thanks for your work!